cmake/AddSYCLExecutable.cmake

```diff
@@ -5,6 +5,14 @@ if (NOT ${SYCL_IMPLEMENTATION} IN_LIST KNOWN_SYCL_IMPLEMENTATIONS)
         "-DSYCL_IMPLEMENTATION=[Intel_SYCL,DPCPP;ComputeCpp,hipSYCL]")
 endif()
 
+if(NOT DEFINED SYCL_CTS_OPENCL_PROXY_CONFIGURED)
```

Comment:
Can't you just do `if(NOT TARGET OpenCL_Proxy)` here?

Reply:
You're right, that is much better! Thanks! 😄